pom.xml

```diff
@@ -308,6 +308,34 @@ limitations under the License.
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>3.1.2</version>
+        <configuration>
+            <configLocation>.mvn/checkstyle.xml</configLocation>
```

Comment:
Consider adding a `<suppressionsLocation>` to do the targeted class/line# "yeah we are intentionally violating style for Reasons" exceptions.